media/css/pocket/includes/_nav.scss

```diff
@@ -2,286 +2,179 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-@import '../utils/variables';
-@import './mobile-nav';
-
+$image-path: '/media/protocol/img';
 $pocket-image-path: '/media/img/pocket';
 
-.pocket-header {
-    width: 100%;
-    background: $color-white;
-    box-shadow: $box-shadow-sm;
-    border-bottom: 1px solid $color-divider;
+@use '../utils/variables' as pocket;
 
-    .global-nav-container {
-        padding: 0 $spacing100;
-        height: 64px;
-        display: flex;
-        align-items: center;
-        max-width: 1168px;
-        margin: 0 auto;
-    }
-}
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/base/elements'; // this probably needs to be in base pocket style imports
```

Comment:
Issue (blocking): I haven't quite pin-pointed the problem, but this seems to be creating a new stacking context somewhere, which is effecting CSS pseudo elements from displaying correctly on the page:

![image](https://user-images.githubusercontent.com/400117/201935860-76408b3d-d973-4aba-be0f-801b24232714.png)